core/src/jvmMain/kotlin/io/kinference/compiler/generation/operators/tensor/GatherGenerator.kt

```diff
@@ -0,0 +1,176 @@
+package io.kinference.compiler.generation.operators.tensor
+
+import io.kinference.compiler.generation.info.TensorInfo
+import io.kinference.compiler.generation.operators.OperatorGenerationInfo
+import io.kinference.compiler.generation.operators.common.TypeAndShapeAwareOperatorGenerator
+import io.kinference.compiler.generation.utils.*
+import io.kinference.ndarray.Strides
+import io.kinference.operators.tensor.Gather
+import kotlin.time.ExperimentalTime
+
+@OptIn(ExperimentalTime::class)
+class GatherGenerator(
```

Comment:
Maybe could you please add links for all operators? (a link to ONNX and Kinference)

Reply:
Good idea, I'll do it.